Solutions/SpyCloud Enterprise Protection/Analytic Rules/SpyCloudEnterpriseProtectionBreachRule.yaml

```diff
@@ -0,0 +1,52 @@
+id: cb410ad5-6e9d-4278-b963-1e3af205d680
+name: SpyCloud Enterprise Breach Detection
+description: |
+  'This alert creates an incident when an malware record is detected in the SpyCloud watchlist data'
+severity: High
+status: Available
+queryFrequency: 12h

```

Comment:
please add the - "requiredDataConnectors: []" property in both the analytic rules. 
Sharing Screenshot for reference - 
![image](https://github.com/Azure/Azure-Sentinel/assets/132428394/d611b72c-4fdf-4bf7-9a09-b5afa4fba267)